parse.py

```diff
@@ -879,6 +892,10 @@ def _handle_field(self, field):
         if type in self._extra_types:
             type_converter = self._extra_types[type]
             s = getattr(type_converter, 'pattern', r'.+?')
+            # -- EXTENSION: group_count attribute
```

Comment:
Similar to my comment above, I don't think we need to commit "extension" notes into the mainline.